test/ctr.bats

```diff
@@ -147,6 +147,43 @@ function check_oci_annotation() {
 	[[ "$output" == "" ]]
 }
 
+@test "ctr pod lifecycle with evented pleg enabled" {
+	ENABLE_POD_EVENTS=true start_crio
+
+	pod_id=$(crictl runp "$TESTDATA"/sandbox_config.json)
+	output=$(crictl pods --quiet)
+	[[ "$output" == "$pod_id" ]]
+
+	ctr_id=$(crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDATA"/sandbox_config.json)
+	output=$(crictl ps --quiet --state created)
+	[[ "$output" == "$ctr_id" ]]
+	wait_for_log "Container created event generated"
```

Comment:
The test is failing because there is no such line in the logs. But I see:
`msg="Container event CONTAINER_CREATED_EVENT generated for 575c19f6a48f25b8d8789ce6b265c0071b3364698260e9e83eb8094984b1fddf" file="server/server.go:971"`

You probably need to update the strings here?